protobuf/syft_proto/types/torch/v1/tensor.proto

```diff
@@ -0,0 +1,28 @@
+syntax = "proto3";
+
+package syft_proto.types.torch.v1;
+
+import "syft_proto/types/syft/v1/id.proto";
+import "syft_proto/types/syft/v1/shape.proto";
+import "syft_proto/types/torch/v1/tensor_data.proto";
+
+message TorchTensor {
+    syft_proto.types.syft.v1.Id id = 1;
+    syft_proto.types.syft.v1.Shape shape = 2;
+    oneof contents {
+        syft_proto.types.torch.v1.TensorData contents_data = 3;
```

Comment:
Why this needs to be separate type?
TensorData also has id and shape, which seems redundant.

Reply:
Repeated fields can't be members of `oneof`, and most of the fields on `TensorData` are repeated. The fields from `TensorData` could be squashed into this type, but then there would be no restriction preventing multiple serialization formats from being set on the same `TorchTensor`.

The id and shape on `TensorData` are a bit redundant, and we might be able to remove them. It seems like the shape should go in the same place as the data, so maybe the best thing to do would be to split the difference, remove shape from `TorchTensor`, and remove id from `TensorData`.

Reply:
@vvmnnnkv Addressed the redundancy in #23.